curvesim/templates/sim_asset.py

```diff
@@ -0,0 +1,28 @@
+from curvesim.constants import Chain
+from curvesim.utils import dataclass
+
+
+@dataclass
+class SimAsset:
+    id: str
+
```

Comment:
`symbol` property would handle the situation where you need the base and quote symbols for the filepath

Reply:
`id` still needed for disambiguation